core/os/android/adb/device.go

```diff
@@ -254,3 +254,53 @@ func parseDevices(ctx context.Context, out string) (map[string]bind.Status, erro
 	}
 	return devices, nil
 }
+
+// NativeBridgeABI returns the native ABI for the given emulated ABI for the
+// device by consulting the ro.dalvik.vm.isa.<emulated_isa>=<native_isa>
+// system properties.
```

Comment:
I think this should also say that it returns the passed in ABI if there is no emulation.

Reply:
never mind... for some reason GH was showing me out of date information and this PR was still open.

Reply:
Still a good point. Will fix shortly.

Reply:
https://github.com/google/gapid/pull/1016/files